packages/popmotion-pose/docs/api/react-native/native-transition.md

```diff
@@ -125,6 +125,16 @@ If `true`, `Transition` will finish animating exiting components out **before**
 
 This can be useful when animating two or more components that affect each other's layout, which can lead to snapping when the exiting components are removed.
 
+### onRest
```

Comment:
Judging by the name I would also expect it to fire after all transition animations are complete - not necessarily only after leaving ones complete. What about entering animations? And moving?

Reply:
Yeah it's a fair point and something I'd considered. However I thought that rerendering only when everything leaving has finished might not turn out to be good enough, and it could be the case we need to start waiting for entering animations too.

However there's a incongruence between the name and the function so I'll earmark it for reconsideration in the next major